contents/newsletter/choosing-technologies.md

```diff
@@ -0,0 +1,158 @@
+
+## Our most common evaluation criteria
+
+Although every problem has its own set of evaluation criteria, there are some recurring themes:
+
+- **Performance:** We ingest billions of events per day and this will only grow. We need technology that can keep up and scale to 100x where we are today. 
+
+- **Cost:** A principle of our pricing is [being the cheapest option](/handbook/how-we-make-money#be-the-cheapest-for-each-individual-product). This means costs are always on our mind and we always need to be searching for ways to reduce them like using S3 for replay storage instead of ClickHouse which helped us [make them drastically cheaper](/blog/session-replay-pricing).
+
+- **Reliability:** Companies rely on us to handle business-critical data. We need to maintain a high uptime and availability. Technologies we choose need to prove they are reliable and handle failures gracefully.
+
+- **Support:** Is this a shiny new thing or a battle-tested one? Is it open source with an active community? We know problems will arise, so we need to know we can get help when we need it. For all of our most important technologies, we have experts we can talk to if we need help.
+
+- **Flexibility:** Because we plan to ship many products beyond what we have today, we need technologies that are interoperable and can be easily adapted to new use cases.
+
+You'll notice that some of these are business factors and others are technical. Many engineering teams would purely focus on the technical factors and rely on product or procurement teams to help evaluate business factors. Our team of [product engineers](/blog/what-is-a-product-engineer) are able to move faster by having the broader business context.
+
+## Make a decision asynchronously
+
+Although individuals are empowered to test and evaluate new technologies, decisions affecting multiple teams are made through a **request for comments** (RFC). These help us get buy-in, gather feedback on a decision, and stay transparent. The process of writing them also forces our teams to clearly articulate their thoughts in a structured way.
+
+An RFC outlines the "why" behind a technology decision and helps facilitate input. The specific parts of an vary, but broadly include the following parts:
+
+- What is the problem you are trying to solve?
+- What are the options you considered?
+- Based on your evaluation, which one do you recommend?
+
+This isn't just a box to tick, it is a core part of helping us choose the right technologies. Our RFC for adopting [Temporal at PostHog](https://github.com/PostHog/meta/pull/99) had 2,594 words, roughly 1/4 of the company as reviewers, 46 comments, and quite a bit of disagreement before finally being adopted. It's now happily in use for [batch exports](/blog/temporal-exports) and [warehouse syncs](/docs/data-warehouse/setup).
```

Comment:
Might be cool to have a screenshot of part of an RFC here, just for a little visual context.